client/src/features/ProjectPageV2/ProjectPageContent/DataConnectors/ProjectConnectDataConnectorsModal.tsx

```diff
@@ -355,3 +406,157 @@ function ProjectLinkDataConnectorBodyAndFooter({
     </Form>
   );
 }
+
+interface DataConnectorDoiFormFields {
+  doi: string;
+}
+
+function ProjectDoiDataConnectorBodyAndFooter({
+  project,
+  toggle,
+}: ProjectConnectDataConnectorsModalProps) {
+  const dispatch = useAppDispatch();
+
+  const [
+    postDataConnector,
+    {
+      data: postDataConnectorData,
+      error: postDataConnectorError,
+      isLoading: postDataConnectorLoading,
+      isSuccess: postDataConnectorSuccess,
+    },
+  ] = usePostDataConnectorsGlobalMutation();
+
+  const [
+    linkDataConnector,
+    {
+      error: linkDataConnectorError,
+      isLoading: linkDataConnectorLoading,
+      isSuccess: linkDataConnectorSuccess,
+    },
+  ] = usePostDataConnectorsByDataConnectorIdProjectLinksMutation();
+
+  const {
+    control,
+    formState: { errors },
+    handleSubmit,
+    reset,
+  } = useForm<DataConnectorDoiFormFields>({
+    defaultValues: {
+      doi: "",
+    },
+  });
+
+  const error = postDataConnectorError ?? linkDataConnectorError;
+  const isLoading = postDataConnectorLoading || linkDataConnectorLoading;
+
+  const onSubmit = useCallback(
+    (values: DataConnectorDoiFormFields) => {
+      postDataConnector({
+        globalDataConnectorPost: {
+          storage: {
+            configuration: {
+              type: "doi",
+              doi: values.doi,
+            },
+            source_path: "/",
+            target_path: "/",
+            readonly: true,
+          },
+        },
+      });
+    },
+    [postDataConnector]
+  );
+
+  // Link the data connector to the project if creation was successful
+  useEffect(() => {
+    if (postDataConnectorSuccess && postDataConnectorData) {
+      linkDataConnector({
+        dataConnectorId: postDataConnectorData.id,
+        dataConnectorToProjectLinkPost: {
+          project_id: project.id,
+        },
+      });
+    }
+  }, [
+    linkDataConnector,
+    postDataConnectorData,
+    postDataConnectorSuccess,
+    project.id,
+  ]);
+
+  // Close the modal and reset the Form if linking was successful
+  useEffect(() => {
+    if (linkDataConnectorSuccess) {
+      dispatch(projectV2Api.util.invalidateTags(["DataConnectors"]));
+      reset();
+      toggle();
+    }
+  }, [dispatch, linkDataConnectorSuccess, reset, toggle]);
+
+  return (
+    <Form noValidate onSubmit={handleSubmit(onSubmit)}>
+      <ModalBody data-cy="data-connector-edit-body">
+        <p className="text-body-secondary">
+          Connect to data on Zenodo, Dataverse, and similar data repositories.
+          More information{" "}
+          <a
+            href="https://renku.notion.site/How-to-connect-data-from-data-repositories-such-as-Zenodo-1eb0df2efafc802ab3bef1c47c8c45b4"
+            rel="noreferrer noopener"
+            target="_blank"
+          >
+            in our documentation
+          </a>
```

Comment:
```suggestion
          <ExternalLink role="link" url={DOI_DOCS}>
            in our documentation
            <BoxArrowUpRight className={cx("bi", "ms-1")} />
          </ExternalLink>
```

with `DOI_DOCS` defined as a constant.

Reply:
Done https://github.com/SwissDataScienceCenter/renku-ui/pull/3656/commits/d7f5651edcec41b894336a3331a86ea3dcbb9f63
I used `iconAfter` and `title` to let the `ExternalLink` component handle the `BoxArrowUpRight` icon